assets/js/componentRequests.js

```diff
@@ -0,0 +1,21 @@
+jQuery(function () {
+    const columns = [
+      {
+        data: 'tf_label',
+        createdCell: function (td, _cellData, rowData) {
+          const display = `${rowData.tf_label}`.trim()
+          $(td).html(`${rowData.name}</a>`)
+        },
+      },
+      {
+        data: 'tf_label',
```

Comment:
This needs to match the data. Have a look at some of the other `/data` endpoint and the table creation in the JavaScript files 🙂

Reply:
Same for other uses of `tf_label`